roles/patch_host_config/tasks/main.yml

```diff
@@ -95,3 +95,34 @@
           ]
         }
       register: http_reply
+
+- name: Load patch for worker partition
+  when: hostvars[host_name]['disks'] is defined
+  vars:
+    disks: "{{ hostvars[host_name]['disks'] }}"
+  block:
+    - name: Template patch partition
+      set_fact:
+        worker_partition: "{{ lookup('template', 'patch-worker-partition.j2') }}"
+
+    - debug: # noqa unnamed-task
+        var: worker_partition
+        verbosity: 1
+  
+  
+    - name: Patch worker partition
+      uri:
+        url: "{{ URL_ASSISTED_INSTALLER_CLUSTER }}/hosts/{{host_id}}/ignition"
```

Comment:
You need to add spaces around host id thats why theres a warning on 113.

Reply:
Thanks ! 
This has been corrected